installer/resources/pacbot_app/files/DB.sql

```diff
@@ -2049,7 +2049,7 @@ INSERT IGNORE INTO cf_RuleInstance (ruleId,ruleUUID,policyId,ruleName,targetType
+INSERT IGNORE INTO cf_RuleInstance (ruleId,ruleUUID,policyId,ruleName,targetType,assetGroup,alexaKeyword,ruleParams,ruleFrequency,ruleExecutable,ruleRestUrl,ruleType,ruleArn,status,userId,displayName,createdDate,modifiedDate,severity,category) VALUES ('VPC_UnCommon_ports_should_not_be_publicly_accessible','VPC_UnCommon_ports_should_not_be_publicly_accessible','VPC_UnCommon_ports_should_not_be_publicly_accessible','UncommonPortsAccess','vpcfirewall','Gcp','VPC_UnCommon_ports_should_not_be_publicly_accessible','{"params":[{"encrypt":false,"value":"check-for-vpc-network-firewall-security-uncommon-ports","key":"ruleKey"},{"key":"port","value":"80,443,20,21,22,23,53,3389,25,465,587,3306,5432,1521,1433,135,137,138,139,445","isValueNew":true,"encrypt":false},{"key":"description","value":"Ensure there are no VPC firewall rules that allow unrestricted inbound access on TCP or udp 1494 and 8010","isValueNew":true,"encrypt":false},{"key":"violationReason","value":"VPC Firewall with uncommon ports was publicly accessible","isValueNew":true,"encrypt":false},{"key":"name","value":"VirtualMachines","isValueNew":true,"encrypt":false},{"key":"policyName","value":"wall with Uncommon ports  should not be publicly accessible","isValueNew":true,"encrypt":false},{"encrypt":false,"value":"critical","key":"severity"},{"encrypt":false,"value":"security","key":"ruleCategory"},{"encrypt":false,"value":"","key":"ruleOwner"}],"environmentVariables":[],"ruleId":"VPC_UnCommon_ports_should_not_be_publicly_accessible","autofix":false,"alexaKeyword":"VPC_UnCommon_ports_should_not_be_publicly_accessible","ruleRestUrl":"","targetType":"vpcfirewall","pac_ds":"gcp","policyId":"VPC_UnCommon_ports_should_not_be_publicly_accessible","assetGroup":"gcp","ruleUUID":"VPC_UnCommon_ports_should_not_be_publicly_accessible","ruleType":"ManageRule"}','0 0/6 * * ? *',NULL,NULL,'ManageRule','arn:aws:events:us-east-1:***REMOVED***:rule/VPC_UnCommon_ports_should_not_be_publicly_accessible','ENABLED','ASGC','Deny Firewall public access to uncommon ports','2022-06-09','2022-06-09','critical','security');
```

Comment:
what has changed here?

Reply:
changed severity Critical to critical in {"encrypt":false,"value":"Critical","key":"severity"}